_source/user-guide/users/single-sign-on/index.md

```diff
@@ -11,17 +11,59 @@ contributors:
   - imnotashrimp
 ---
 
-Single sign-on (SSO) allows you to manage access to your Logz.io account from a single user and identity management tool.
+Single sign-on (SSO) allows you to manage access to your Logz.io account
+from a single user and identity management tool.
 
 ## What is SSO?
 
-SSO is a way to authenticate and sign in to Logz.io with the same credentials that you use for different services.
-With SSO, users won't need remember different usernames and passwords for each service, and you can manage access to all your services from one service.
-Enabling SSO on your account is more convenient for your users and more secure for your company.
+SSO is a way to authenticate and sign in to Logz.io
```

Comment:
I would write the real meaning of SSO (as an answer to the 'what is SSO' question)
It's bigger than only be the way to authenticate for Logz.io :)